test/TestSites/web.config

```diff
@@ -2,7 +2,7 @@
   <system.webServer>
     <!-- This repository cannot use the iis publish tool because there's a bug in the installer if the tool package is a package reference -->
     <!-- See https://github.com/dotnet/cli/issues/1215 -->
-    <aspNetCore forwardWindowsAuthToken="true" processPath="..\TestSites.exe" stdoutLogEnabled="false" stdoutLogFile="..\logs\stdout.log" startupTimeLimit="3600" />
+    <aspNetCore forwardWindowsAuthToken="true" processPath=".\TestSites.exe" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout.log" startupTimeLimit="3600" />
```

Comment:
Without it IIS cannot launch. I believe there is an issue filed for that. https://github.com/aspnet/IISIntegration/issues/119